src/Hanselman/AppShell.xaml.cs

```diff
@@ -1,12 +1,25 @@
-﻿using Xamarin.Forms;
+﻿using Hanselman.Views;
+using Xamarin.Forms;
 
 namespace Hanselman
 {
     public partial class AppShell : Shell
     {
+        public const string PodcastDetails = "sections/tab_podcasts/podcasts/podcast_details";
```

Comment:
These all don't have to have full paths you can just have them be things like this
```c#
public const string PodcastDetails = "PodcastDetails";
public const string PodcastEpisode = "PodcastEpisode";
...
```

Reply:
Is there any advantage to doing the full paths though? Deep linking? or anything like that?

Else, is just doing nameof() on pages good?